### PR TITLE
Add firebase analytics

### PR DESCRIPTION
**Pull request by antho1404**
Created a firebase analytics plugin as it doesn't exist yet.
This should track all page views of the documentation and the documentation is already hosted on firebase. The tracking is already deployed on production too

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
   "scripts": {
     "dev": "vuepress dev --no-clear-screen",
     "build": "vuepress build",
-    "vuepress": "vuepress"
+    "vuepress": "vuepress",
+    "deploy:prod": "npm run build && firebase deploy --only hosting:documentation-liteflow",
+    "deploy:staging": "npm run build && firebase deploy --only hosting:docs-liteflow-staging"
```
Comment (NicolasMahe) [inline package.json:9]: firebase should not be in the scripts as it is not a dependency and it is not related to the source code, it's a env related stuff.

File: firebase.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "hosting": {
+    "site": "documentation-liteflow",
```
Comment (NicolasMahe) [inline firebase.json:3]: there is nothing for "docs-liteflow-staging"

Comment (antho1404) [reply firebase.json:3]: Yes, I'm not sure yet how this works. It seems that I need a unique name but I don't know if I can set multiple environments here

Comment (NicolasMahe) [reply firebase.json:3]: https://firebase.google.com/docs/hosting/multisites#define_hosting_config

**Pull request merged**